pilot/model/context.go

```diff
@@ -12,10 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package proxy
+package model
```

Comment:
Also, why this move? There is nothing generic here to be moved into the model. The environment differs from platform to platform right/

Reply:
ignore please